model/registry/cicd.yaml

```diff
@@ -0,0 +1,198 @@
+groups:
+  - id: registry.scm
+    prefix: scm
+    type: attribute_group
+    brief: >
+     This group defines the attributes for source control management (SCM) systems.
```

Comment:
this group defines nothing. most probably you should move other fields under this group